### PR TITLE
Implement event forms

### PR DESCRIPTION
**Pull request by botronic**
Adds support for filling in event forms on the web. The indicator icons now work and the feedback generation fields are pre populated with data from backend.

### CODE REVIEW

**Comment by mictab** [comment]
Still wip or ready for review? @botronic

**Comment by botronic** [comment]
Its ready!!!!!!!!!!!!!!!!!!! @mictab

File: app/api.js [hunk 1]
```diff
@@ -52,6 +52,23 @@ function ftch(...args) {
     .then(parseJSON)
 }
 
+// function executeGraphQLWithHeader(query, header) {
```
Comment (mictab) [inline app/api.js]: Remove commented code.

File: app/components/EventFeedbackForm/template.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    title: 'Are you familiar with the company and what they do?',
+    type: 'choice',
+    labels: ['Yes.', 'To some extent.', 'No.'],
+    labelValues: ['YES', 'SOMEWHAT', 'NO'],
+    name: 'familiarWithCompany',
+  },
+  {
+    title: 'How interested in working at this company are you?',
+    type: 'scale',
+    labels: ['Not interested at all.', 'Very interested.'],
```
Comment (mictab) [inline app/components/EventFeedbackForm/template.js]: Should be "Not at all interested"?

File: app/components/EventFeedbackForm/template.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    name: 'viewOfCompany',
+  },
+]
+
+export const postEventQuestions = [
+  {
+    title: 'How did the event impact your opinion about the company?',
+    type: 'choice',
+    labels: [
+      'The event had a positve impact on my view of the company.',
```
Comment (mictab) [inline app/components/EventFeedbackForm/template.js]: positve -> positive

File: app/components/EventFeedbackForm/template.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      "It didn't change my view of the company.",
+      'The event had a negative impact on my view of the company.',
+    ],
+    labelValues: ['POSITIVE', 'NEUTRAL', 'NEGATIVE'],
+    name: 'eventImpact',
+  },
+  {
+    title: 'How interested in working at this company are you now?',
+    type: 'scale',
+    labels: ['Not interested at all.', 'Very interested.'],
```
Comment (mictab) [inline app/components/EventFeedbackForm/template.js]: Not at all interested?

**Pull request merged**